yrs/src/store.rs

```diff
@@ -26,10 +27,6 @@ pub(crate) struct Store {
     /// operations) integrated - and therefore visible - into a current document.
     pub(crate) blocks: BlockStore,
 
-    /// Handles subscriptions for the `afterTransactionCleanup` event. Events are called with the
-    /// newest updates once they are committed and compacted.
-    pub(crate) after_transaction_events: Box<Option<EventHandler<AfterTransactionEvent>>>,
```

Comment:
Something I've missed in @Waidhoferj PR last time - don't do `Box<Option<T>>`: it's useless. We always allocate this way. Reverse (`Option<Box<T>>`) allocates only when field is used. Additionally: `struct EventHandler(Box<HashMap>)` already boxes necessary content, so there's no need to double boxing.

Reply:
Got it, thanks for the feedback!